packages/node/src/node.ts

```diff
@@ -241,25 +241,34 @@ export class Node {
             params.multisigAddress,
             context.finalCommitment!
           );
-        } else {
-          if (!context.commitments[0]) {
-            throw new Error(
-              `State transition without commitment: ${JSON.stringify(message)}`
+        } else if (
+          protocol === Protocol.Install ||
+          protocol === Protocol.Uninstall ||
+          protocol === Protocol.Update
+        ) {
+          /// install, update, uninstall,
+          /// All other protocols are stored under the key
+          /// appIdentityHash/protocolName. This means that an app only has one
+          /// update commitment.
+          const params = message.params as any;
```

Comment:
Don't cast to `any`

Reply:
Typescript doesn't let me access `params.multisigAddress` without the cast

Reply:
Cast it to what it is, not `any`. In this case it would be `SetupParams | InstallParams | UninstallParams`